lib/web_console/railtie.rb

```diff
@@ -37,6 +37,9 @@ class Railtie < ::Rails::Railtie
     end
 
     initializer 'web_console.mount_point' do
+      if root = Rails.application.config.relative_url_root
```

Comment:
> Can we call root.join(Middleware.mount_point) below?

It seems the `root` is passed as a string. At first, I have tried to use `Pathname`, but the mount point could be like an absolute path, and it could ignore the relative root by `Pathname.join`. So I think that `File.join` works well than `Pathname.join` in this case.

> Also, can we test what happens if both config.relative_url_root and config.web_console.mount_point are set?

Yes, we can 👍